### PR TITLE
local_addr

### PR DESCRIPTION
**Pull request by mzsombor**

### CODE REVIEW

File: aiosnmp/connection.py [hunk 1]
```diff
@@ -27,11 +29,15 @@ def __init__(
         *,
         host: str,
         port: int = 161,
+        local_addr: str = None,
```
Comment (hh-h) [inline aiosnmp/connection.py:32]: let's use `local_addr: Optional[Tuple[str, int]] = None` and remove `local_port` at all.

**Comment by hh-h** [comment]
LGTM, something wrong with tests in docker, I'll investigate later.

**Comment by mzsombor** [comment]
> LGTM, something wrong with tests in docker, I'll investigate later.

FYI, I'm more of a network engineer than a programmer so feel free to change if you think it is required. Thank you again.

**Comment by hh-h** [comment]
That's totally fine! Something happened with tests for ipv6 and localhost, I'll investigate when I have spare time and release a new release, thanks!

**Pull request merged**